### PR TITLE
Refactor code with rules

### PR DESCRIPTION
**Pull request by audrius-savickas**
Added configuration file. Refactored the whole project. Changed all types to implicit in frontend code. Renamed functions to capital case.

### CODE REVIEW

**Review by DonatasPuc — APPROVED** [review]

**Comment by DonatasPuc** [comment]
It would be great if you looked through the changes and follow this coding style in the future.

File: wasted-app/Model/Repositories/BaseRepository.cs [hunk 1]
```diff
@@ -11,39 +11,39 @@ public abstract class BaseRepository<T> : IBaseRepository<T> where T : BaseEntit
...
-        public BaseRepository(String pathToDatabase)
+        public BaseRepository(string pathToDatabase)
         {
             _pathToDatabase = pathToDatabase;
         }
 
         public void Add(T entity)
         {
-            List<T> all = GetAll().ToList();
+            var all = GetAll().ToList();
```
Comment (audrius-savickas) [inline wasted-app/Model/Repositories/BaseRepository.cs:21]: was backend code supposed to be refactored to vars only?

Comment (DominykasSip) [reply wasted-app/Model/Repositories/BaseRepository.cs:21]: yeah I was wondering that too

**Review by audrius-savickas — COMMENTED** [review]
seems good except that backend code was refactored to vars, althought we didn't want to?

**Comment by DominykasSip** [comment]
Everything looks good regarding the refractor, but why did the backend got its types redone to vars?

**Comment by DonatasPuc** [comment]
Yeah, I somehow messed up refactoring implicit types.

**Comment by DonatasPuc** [comment]
I think I fixed it.

**Review by audrius-savickas — COMMENTED** [review]
looks great now

**Review by JesusGonzalezA — APPROVED** [review]

**Pull request merged**